atomicfu/src/nativeTest/kotlin/kotlinx/atomicfu/test/SynchronizedTest.kt

```diff
@@ -0,0 +1,38 @@
+package kotlinx.atomicfu.test
+
+import kotlinx.atomicfu.*
+import kotlin.native.concurrent.*
+import kotlin.native.concurrent.AtomicInt
+import kotlin.test.*
+
+class SynchronizedTest {
+
+    private val iterations = 1000
+    private val COUNT = 20
```

Comment:
Please, give it a more descriptive name. It is count of what? `nWorkers` would be a better name, for example.